sys/arm64/arm64/machdep.c

```diff
@@ -473,8 +530,13 @@ get_mcontext(struct thread *td, mcontext_t *mcp, int clear_ret)
 		mcp->mc_gpregs.gp_spsr = tf->tf_spsr;
 	}
 
+#if !__has_feature(capabilities)
+	for (int i = 1; i < nitems(mcp->mc_gpregs.gp_x); i++)
+		mcp->mc_gpregs.gp_x[i] = tf->tf_x[i];
+#else
 	memcpy(&mcp->mc_gpregs.gp_x[1], &tf->tf_x[1],
 	    sizeof(mcp->mc_gpregs.gp_x[1]) * (nitems(mcp->mc_gpregs.gp_x) - 1));
+#endif
 
 	mcp->mc_gpregs.gp_sp = tf->tf_sp;
 	mcp->mc_gpregs.gp_lr = tf->tf_lr;
```

Comment:
You probably want to save `ddc` here and restore it in `set_mcontext`?

Reply:
Yes, in a future change we will.